sentimentanalysis/analyzer.py

```diff
@@ -111,7 +113,7 @@ class TweetSentiment(object):
     Gives an intensity score to the inputted tweet.
     """
 
-    def __init__(self, text, lexicon_file="sentimentanalysis/lexicon_done.txt"):
+    def __init__(self, text, lexicon_file="lexicon_done.txt"):
```

Comment:
I don't get this... why does it sometimes work.

We can just keep in mind that sometimes you might need 'sentimentanalysis/' sometimes not???